src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -320,6 +313,33 @@ public void checkifDownloadDirectoryisEmpty() throws IOException {
         assertFalse(Hooks.getDownloadDirectory().list().length > 0, "directory is not empty");
     }
 
+    /**
+     * Assert filename ends with date and time stamp.
+     * @param dateTimeFormat expected Date and Time
+     * @param fileName expected File name
+     */
+    @Then("^the \"([^\"]*)\" file ends with \"([^\"]*)\"$")
+    public void surveillanceActivityFileEndsWithDateTimeFormat(final String fileName, final String dateTimeFormat) {
+        File[] filenames = Hooks.getDownloadDirectory().listFiles();
+        for (File file : filenames) {
+            String dwldFileName = file.getName();
+            if (dwldFileName.startsWith(fileName)) {
+                String[] fileTokens = dwldFileName.split("-");
+                String extToken = fileTokens[fileTokens.length - 1];
+                String[] dTokens = extToken.split("\\.");
+                String fileDate = "-" + dTokens[0];
+                SimpleDateFormat df = new SimpleDateFormat(dateTimeFormat);
```

Comment:
The dateTimeFormat value that's passed in is `-yyyMMdd_HHmmss`, including that initial `-`. Does that get correctly converted to a SimpleDateFormat? Even if it does work, it shouldn't